src/fmeval/eval_algorithms/qa_accuracy.py

```diff
@@ -17,6 +16,7 @@
 from fmeval.eval_algorithms.util import (
     validate_dataset,
     get_dataset_configs,
+    normalize_text_quac_protocol,
```

Comment:
Good idea to move shared functions out, let's do more of this as we continue to modularize the code base. I suggest we add a `metrics.py` in addition to `utils.py` for all metric-related computations going forward.